README.md

```diff
@@ -0,0 +1,20 @@
+# Codacy Docker Base
+
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/0cd6b6dd0d60416bbca459149dd9f18a)](https://www.codacy.com/app/Codacy/base?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=codacy/base&amp;utm_campaign=Badge_Grade)
+[![CircleCI](https://circleci.com/gh/codacy/base.svg?style=svg)](https://circleci.com/gh/codacy/base)
+[![Docker Version](https://images.microbadger.com/badges/version/codacy/base.svg)](https://microbadger.com/images/codacy/base "Get your own version badge on microbadger.com")
+
+## Contents
+
+Dockerfile describing a docker containing:
+- Ubuntu 18.04.*
+- OpenJDK 8 JRE
+- Docker cli 17.09.*
+- Git cli 2.*
+- Unix tools: curl, wget, unzip and vim.
+
+## Build and Publish
+
+```sh
+make help
+```
```

Comment:
add the "normal" piece about Codacy itself

Reply:
done, thanks